pootle/apps/contact/forms.py

```diff
@@ -56,6 +57,17 @@ def from_email(self):
     def recipient_list(self):
         return [settings.POOTLE_CONTACT_EMAIL]
 
+    def save(self, fail_silently=False):
+        """
+        Build and send the email message.
+
+        """
+        kwargs = self.get_message_dict()
+        kwargs["headers"] = {"reply-to": kwargs["from_email"]}
+        kwargs["from_email"] = settings.DEFAULT_FROM_EMAIL
```

Comment:
well - for me it kinda depends on context - but here where we are capturing an existing value and setting to a different key - and then updating the original key - i think its clearer to this step-by-step